packages/clay-form/docs/input-group.mdx

```diff
@@ -167,6 +167,48 @@ export default function App() {
 }
 ```
 
+### Inset Before
+
+```jsx preview
+import {Provider} from '@clayui/core';
+import Form, {ClayInput} from '@clayui/form';
+import ClayButton from '@clayui/button';
+import ClaySticker from '@clayui/sticker';
+import React, {useState} from 'react';
+
+import '@clayui/css/lib/css/atlas.css';
+
+export default function App() {
+	return (
+		<Provider spritemap="/public/icons.svg">
+			<div className="p-4">
+				<Form.Group>
+					<label htmlFor="selectASpace">Select a Space</label>
+					<ClayInput.Group>
+						<ClayInput.GroupItem>
+							<ClayInput
+								aria-label="Select a space"
+								className="form-control-select"
+								id="selectASpace"
+								insetBefore
+								placeholder="Select a Space"
+								type="text"
+								value="Space Program"
+							/>
+							<ClayInput.GroupInsetItem before>
```

Comment:
@pat270 Is there a reason why this doesn't come before the input? It seems odd since it's an before insetItem.